.travis.yml

```diff
@@ -1,29 +1,42 @@
 language: C
 sudo: false
 dist: trusty
-# We can't use strict flags in CFLAGS as a general environment
-# variable, because that messes up ./configure.
-script: make CFLAGS="$CFLAGS $CFLAGS_STRICT" && make test USE_VALGRIND=1
+addons:
+  apt:
+    packages:
+      - valgrind
 matrix:
   include:
     - compiler: gcc
-      env: CFLAGS="-std=c99 -O2 -Wall -Wextra -Werror"
-           CFLAGS_STRICT="-Wpedantic -pedantic-errors -Wno-clobbered"
+      env: CFLAGS_STRICT="-Wall -Wextra -Werror -Wpedantic -pedantic-errors -Wno-clobbered"
     - compiler: clang
-      env: CFLAGS="-std=c99 -O2 -Wall -Wextra -Werror"
-           CFLAGS_STRICT="-Weverything -Wno-#warnings -Wno-pedantic -Wno-padded -Wno-format-nonliteral -Wno-disabled-macro-expansion -Wno-missing-noreturn -Wno-reserved-id-macro -Wno-cast-align"
+      env: CFLAGS_STRICT="-Wall -Wextra -Werror -Weverything -Wno-#warnings -Wno-pedantic -Wno-padded -Wno-format-nonliteral -Wno-disabled-macro-expansion -Wno-missing-noreturn -Wno-reserved-id-macro -Wno-cast-align"
 env:
   global:
-    - secure: "Lc/FFVefKMmvzxQ0Tv6fB64nZbby6slUdjSxuIVg8cvSfQh8I3J9AN6fRDcDZ1VB5JLIdyddNNQvcMb4qFS08qp1/Q24uLYRC+vmzkdCY5dIJKBAzpmm6i35UTdTdQG1D1g3B8K1jwQl5CdSj9JFHThaxlNC3y2GDFzs1LGQpIHYrEZ8bMBk2wh8mYnggXagQheGp7RMsIMisdDzq2mzd5gR2w+1oGLAearGfY1MpxfQm8fkk7pIWNnpag9oQ5p16UvHm3A1r4RLsaZVJR9iwkYDwma6wXsU26vBkixX9D5ZCloql55u/HCK+GKm2wTHSuwtKnDup2tTarSao+fMfVEEF8p4GCMQZSffhb7mVcUO7qjiKt+Uod45EkxUxRf02PKsSWU8m078cycNwz5/+vpg2or/T/JIn2g91OKbbufPXtH9LHFD+YTLlYqZjvfwF79Ea9st8JRomJ2N+mc9cqaXoB2r4ckzJrtdxamoSjibIQa1gjM0TT2jyvkOqCFDkePVaNXbhOxq1lwAPnlKoMV3yXPUKytEo9a9ug6y+SQUFzHzRgbF67KV2Jb7r/J7lTkaB15fxZQ8aJhugcG7vLRUkT239vj4TciRmrMrjfVrIWONs1B3QI0G0C+clV3TTHWosFNrA1F4FmOrNUYb7hnzd1dGZIfHrGQrmKiabs8="
+    # Some projects need -std=c99 to specify library support for ./configure
```

Comment:
So far, this only applies to Tarsnap and wcslen(), but there's no harm in standardizing on this.